website/src/components/SearchPage/SearchFullUI.tsx

```diff
@@ -84,6 +84,20 @@ export const InnerSearchFullUI = ({
 
     const [isColumnModalOpen, setIsColumnModalOpen] = useState(false);
 
+    const columnFieldItems: FieldItem[] = useMemo(
+        () =>
+            schema.metadata
+                .filter((field) => !(field.hideInSearchResultsTable ?? false))
+                .map((field) => ({
+                    name: field.name,
+                    displayName: field.displayName ?? field.name,
```

Comment:
It would be nice to have the same displayName for each field in the search and in the download options box, in search we convert displayName additionally using 
`displayName: filter.displayName ?? filter.label ?? sentenceCase(filter.name),`
this means that accessionVersion becomes Accession version in search but stays accessionVersion in the download box.

Reply:
Thanks - I agree we should make this consistent. I think the use of `sentenceCase` on a technical name is really not a good pattern and the right way to do this is to remove the `sentenceCase` and to add `displayNames` for everything. Since the current state is not a regression, just maintains current behaviour, I will make an issue for this. (We already have more displayNames on PPX so this looks better there, but we need more).

Reply:
https://github.com/loculus-project/loculus/issues/4089

Reply:
(For an example of how we need the display names anyway, see the review cards:

<img width="296" alt="image" src="https://github.com/user-attachments/assets/4582c6f7-4527-4012-b111-3f3f635ee404" />


)